base.js

```diff
@@ -7,32 +7,35 @@ import { Configuration, OpenAIApi } from "openai";
 import ora from "ora";
 import { globby } from "globby";
 import prettier from "prettier";
-
+const pkg = JSON.parse(await fs.readFile("package.json"));
 config();
 
 const { flags } = meow(
 	`
 Usage
-  $ node <start generation>
+  $ node <starter file>
 
 Options
-  -G, --goal           Set the goal of the generated code. Default is "extend the code".
+  -p, --prompt         Set the main prompt of the generated code. Default is "extend the code".
   -g, --generations    Set the number of generations for the generated code. Default is 1.
-  -p, --persona        Set the persona of the generated code. Default is "expert node.js developer, creative, code optimizer, interaction expert".
+  -P, --persona        Set the persona of the generated code. Default is "expert node.js developer, creative, code optimizer, interaction expert".
   -t, --temperature    Set the temperature for the generated code. Default is 0.2.
-  -c, --clean          Set to true if you want to remove any previously generated code.
+  -c, --clean          Set to true if you want to remove any previously generated code.  Default is false.
   -m, --model          Set the model to use for generating the code. Default is "gpt-3.5-turbo".
+  -n, --negativePrompt Set the negative prompt. Default is "".
+  -s, --seed           Set the seed. Default is -1.
 
 Examples
-  $ node generation-000.js -G "console based mandelbrot set ascii" -g 3 -p "Node.js developer, creative" -c
+  $ node generation-000.js -p "mandelbrot set" -g 3 -P "Node.js developer, creative" -c
```

Comment:
Get rid of the persona, as we already have those as the default

```suggestion
  $ node generation-000.js -p "mandelbrot set" -g 3 -c
```

Reply:
this is valid since it showcases a persona that is different from the default